task2/src/main/java/com/epam/shop/container/ListContainer.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.shop.container;
+
+import com.epam.shop.model.Vehicle;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.function.Consumer;
+
+public class ListContainer<T extends Vehicle> extends Container  {
```

Comment:
Why is this in separated class, why dont you make it to be inner class of container